minmax_stack.rb

```diff
@@ -4,37 +4,60 @@
 # "maximum" _and_ "minimum" operations.
 
 
+
 class MinMaxStack
+
+  class DimensionError < StandardError;end
+
+  attr_accessor :items
+
   def initialize
+    @list = LinkedList.new
+    @items = []
```

Comment:
There's no reason to use an `Array` here — you could just use another `LinkedList` with `unshift` instead of `push`.